app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/dialogs/SendGeographicMessageDialog.java

```diff
@@ -125,8 +140,14 @@ protected boolean hasPositiveButton() {
     protected void onPositiveClick() {
         sLogger.userInteraction("Clicked OK");
 
-        mMessageSender.sendLatLongMessageAsync(mPoint);
-
+        EditText editText = (EditText) getDialog().findViewById(R.id.dialog_send_message_edit_text);
```

Comment:
View finding should be done at fragment creation (use ButterKinfe).
Otherwise, each click on the positive button will look for the view inside the layout, which is wasteful. !fix